frontend/src/components/maps/leaflet/InfoSlideOut/ThreeColumnItem.tsx

```diff
@@ -28,7 +28,7 @@ const RightCol = styled(Col)`
 
 interface IThreeColItem {
   leftSideLabel: string;
-  rightSideItem: string | number | undefined;
+  rightSideItem: string | number | React.ReactNode | undefined;
```

Comment:
So the new `ProjectNumberLink` component can be used

Reply:
Good article about passing components around that helped me:
https://medium.com/better-programming/typescript-reactjs-the-element-vs-reactelement-vs-htmlelement-vs-node-confusion-6cda21315ddd

Reply:
Thanks! Will give it a read